packages/amplify_test/lib/test_models/Address.dart

```diff
@@ -8,7 +8,7 @@
 // ignore_for_file: public_member_api_docs, annotate_overrides, dead_code, dead_codepublic_member_api_docs, depend_on_referenced_packages, file_names, library_private_types_in_public_api, no_leading_underscores_for_library_prefixes, no_leading_underscores_for_local_identifiers, non_constant_identifier_names, null_check_on_nullable_type_parameter, prefer_adjacent_string_concatenation, prefer_const_constructors, prefer_if_null_operators, prefer_interpolation_to_compose_strings, slash_for_doc_comments, sort_child_properties_last, unnecessary_const, unnecessary_constructor_name, unnecessary_late, unnecessary_new, unnecessary_null_aware_assignments, unnecessary_nullable_for_final_variable_declarations, unnecessary_string_interpolations, use_build_context_synchronously, implicit_dynamic_parameter, implicit_dynamic_map_literal, implicit_dynamic_type
 
 import 'package:amplify_core/amplify_core.dart';
-import 'package:flutter/foundation.dart';
+import 'package:meta/meta.dart';
```

Comment:
What is the reason for this change? I think codegen uses includes `import 'package:flutter/foundation.dart';`

Reply:
Oh I see. You are removing flutter as a dep from this package. I think the change is fine. Might be a little confusing if we regenerate these but not a big deal.

Reply:
Yeah, exactly. We can move test models to their own package if needed or add a `amplify_flutter_test` pkg as well if this becomes too cumbersome.

Hoping codegen gets fixed first though